quipucords/scanner/network/inspect.py

```diff
@@ -139,9 +139,22 @@ def inspect_scan(self):
 
         :returns: An array of dictionaries of facts
         """
-        roles = ['check_dependencies', 'connection', 'cpu', 'date',
-                 'dmi', 'etc_release', 'file_contents', 'jboss_eap',
-                 'ifconfig', 'uname', 'virt', 'virt_what', 'host_done']
+        roles = [
```

Comment:
FWIW, I exploded this into multiple lines because otherwise we're going to thrash the commit history as we add to the list.